pint/testsuite/pandas_test_interface.py

```diff
@@ -334,10 +380,15 @@ def test_pintarray_creation(self):
         for y in ys:
             self.assertQuantityAlmostEqual(x,y.data)
 
+    
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```

Comment:
I'd like to keep these warnings popping up unless there's a good reason to silence them?